content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -6,12 +6,19 @@ const {GroupedActivityFeed} = require("components/ActivityFeed/ActivityFeed");
 const Spotlight = require("components/Spotlight/Spotlight");
 const Search = require("components/Search/Search");
 const {actions} = require("actions/action-manager");
+let REPORT_RENDER_EVENT = true;
 
 const NewTabPage = React.createClass({
   // TODO: Replace with real search api via addon
   onSearch(value) {
     this.props.dispatch(actions.NotifyPerformSearch(value));
   },
+  componentWillUpdate(nextProps, nextState) {
+    if (nextProps.isReady && REPORT_RENDER_EVENT) {
+      this.props.dispatch(actions.NotifyTelemetry("NEWTAB_RENDER"));
```

Comment:
Asking the question again. I'm curious. When would NEWTAB_RENDER fire multiple times?
You set a global at line 11 and disable it at line 21